target/linux/qualcommbe/ipq95xx/base-files/lib/upgrade/platform.sh

```diff
@@ -9,6 +9,7 @@ platform_check_image() {
 
 platform_do_upgrade() {
 	case "$(board_name)" in
+	alta,route10\
```

Comment:
This should be sorted in alphabetical order:
```
	8devices,kiwi-dvk|\
	alta,route10)
```

Reply:
list re-ordered